src/components/VipMetadataOverlay.vue

```diff
@@ -20,19 +22,23 @@ export default defineComponent({
   props: {
     patientName: {
       type: String,
-      required: false
-    }, 
+      required: false,
+      default: ''
+    },
     patientBirthdate: {
       type: String,
-      required: false
-    }, 
+      required: false,
+      default: ''
+    },
     institutionName: {
       type: String,
-      required: false
-    }, 
+      required: false,
+      default: ''
+    },
     instanceCreationDateTime: {
       type: String,
-      required: false
+      required: false,
+      default: ''
```

Comment:
Defining default values for these parameters seems a bit redundant to me because if the value is empty, it will be overwritten by the corresponding place holders (patientNameNotDefined, birthdateNotDefined, institutionNameNotDefined, etc.)

Reply:
since `required: false`, it requires a default value as a fallback. So, maybe use placeholder as a default value
```js
    instanceCreationDateTime: {
      type: String,
      required: false,
      default: $gettext('instance creation date and time not defined')
```

Reply:
If the solution suggested by @saw-jan doesn't work just leave it as it is with `default: ''`